grails-app/views/purchaseOrder/_enterOrderDetails.gsp

```diff
@@ -88,6 +88,7 @@
                                         <g:selectLocation class="chzn-select-deselect filter"
                                                           id="destination.id"
                                                           name="destination.id"
+                                                          hasOrganization="${true}"
```

Comment:
I think `withOrganization` would sound better? :thinking:  but since I'm not master of naming, treat it as a loose suggestion